pkg/controller/cronjob/utils.go

```diff
@@ -173,8 +173,8 @@ func getRecentUnmetScheduleTimes(sj batchv1beta1.CronJob, now time.Time) ([]time
 func getJobFromTemplate(sj *batchv1beta1.CronJob, scheduledTime time.Time) (*batchv1.Job, error) {
 	// TODO: consider adding the following labels:
 	// nominal-start-time=$RFC_3339_DATE_OF_INTENDED_START -- for user convenience
-	// scheduled-job-name=$SJ_NAME -- for user convenience
 	labels := copyLabels(&sj.Spec.JobTemplate)
+	labels["cronjob-name"] = sj.ObjectMeta.Name
```

Comment:
I would vote for namespacing labels that Kubernetes adds into resources.

Reply:
How about something similar to what we have in [Deployments](https://github.com/kubernetes/kubernetes/blob/11104d75f15e43ac41aa5d7226837f8a81523572/pkg/controller/deployment/util/deployment_util.go#L52)?

`cronjob.kubernetes.io/name` ?